src/logging.js

```diff
@@ -80,23 +81,23 @@ function getTransports(json, isDev) {
 type Loggers = {
     default: Logger,
     middleware: {
-        requestLogger: any,
-        errorLogger: any,
+        requestLogger: Middleware,
+        errorLogger: Middleware,
     },
 };
 
 function initLogging(logLevel: LogLevel, isDev: boolean): Loggers {
     // This is the logger that captures requests handled by our express server.
     const requestLogger = expressWinston.logger({
-        level: logLevel,
+        level: `${logLevel}`,
```

Comment:
I wish `$Keys<T>` implicitly mapped to `string`, but it doesn't....so I did this.